astropy/modeling/utils.py

```diff
@@ -402,8 +402,9 @@ def validate(cls, model, bounding_box):
                    "containing such a sequence as its sole element.").format(
                        model.name)
 
-            assert (isiterable(bounding_box) and
-                        np.shape(bounding_box) in ((2,), (1, 2))), msg
+            if (not isiterable(bounding_box)
```

Comment:
While you're at it, could you move the definition of `msg` inside the error clause here? Seems silly to define it if it doesn't get used!